Release to master

ioncreature opened the pull request:
Release

ioncreature committed:
Merge pull request #2811 from energywebfoundation/master

Release

alexworker23 reviewed (changes requested):
PR from `release` to `master` should include versions update

github-actions[bot] committed:
chore(release): publish /skip-deploy

 - @energyweb/device-registry@8.0.2
 - @energyweb/origin-backend-client@10.0.4
 - @energyweb/origin-backend-core@8.0.2
 - @energyweb/origin-backend@11.0.0
 - @energyweb/origin-backend-app@1.5.2
 - @energyweb/origin-backend-irec-app@1.2.0
 - @energyweb/origin-ui@2.6.0
 - @energyweb/origin-device-registry-api-client@0.0.4
 - @energyweb/origin-device-registry-api@0.2.0
 - @energyweb/origin-device-registry-irec-form-api-client@1.0.3
 - @energyweb/origin-device-registry-irec-form-api@2.0.1
 - @energyweb/origin-device-registry-irec-local-api-client@0.2.0
 - @energyweb/origin-device-registry-irec-local-api@0.3.0
 - @energyweb/origin-energy-api-client@1.1.1
 - @energyweb/origin-energy-api@1.0.3
 - @energyweb/origin-organization-irec-api-client@0.2.0
 - @energyweb/origin-organization-irec-api@1.5.0
 - @energyweb/migrations-irec@1.2.0
 - @energyweb/migrations@3.2.5
 - @energyweb/solar-simulator@1.2.12
 - @energyweb/issuer-api-client@0.2.0
 - @energyweb/issuer-api@0.3.0
 - @energyweb/issuer-irec-api-client@0.2.0
 - @energyweb/issuer-irec-api-wrapper@0.5.0
 - @energyweb/issuer-irec-api@0.2.0
 - @energyweb/issuer@4.0.0
 - @energyweb/exchange-client@0.1.4
 - @energyweb/exchange-core-irec@1.0.3
 - @energyweb/exchange-core@3.2.1
 - @energyweb/exchange-io-erc1888@1.2.0
 - @energyweb/exchange-irec-client@0.0.4
 - @energyweb/exchange-irec@1.0.3
 - @energyweb/exchange-token-account@0.2.0
 - @energyweb/exchange@1.10.0
 - @energyweb/exchange-ui-core@1.4.1
 - @energyweb/localization@0.15.0
 - @energyweb/origin-ui-core@1.14.1
 - @energyweb/origin-ui-irec-core@0.2.0
 - @energyweb/origin-backend-utils@1.6.0
 - @energyweb/utils-general@11.0.3

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/energywebfoundation/origin?pullRequest=2813) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/energywebfoundation/origin?pullRequest=2813) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: ioncreature<br/>:x: github-actions[bot]<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/energywebfoundation/origin?pullRequest=2813) it.</sub>

ioncreature committed:
fix(migrations): fixed dockerfiles

github-actions[bot] committed:
chore(release): publish /skip-deploy

 - @energyweb/migrations@3.2.6

ioncreature committed:
Merge branch 'master' into release

# Conflicts:
#	packages/ui/exchange-ui-core/package.json

ioncreature committed:
fix(migrations): stable dockerfile fix

ioncreature committed:
fix(migrations): stable dockerfile fix

ioncreature committed:
Merge remote-tracking branch 'origin/release' into release

github-actions[bot] committed:
chore(release): publish /skip-deploy

 - @energyweb/origin-backend-app@1.5.3
 - @energyweb/origin-ui@2.6.1
 - @energyweb/migrations@3.2.7
 - @energyweb/exchange-ui-core@1.4.2
 - @energyweb/origin-ui-core@1.14.2
 - @energyweb/origin-ui-irec-core@0.2.1

ioncreature committed:
fix(migrations): stable dockerfile fix

ioncreature committed:
Merge remote-tracking branch 'origin/release' into release

github-actions[bot] committed:
chore(release): publish /skip-deploy

 - @energyweb/origin-backend-app@1.5.4
 - @energyweb/origin-ui@2.6.2
 - @energyweb/migrations@3.2.8
 - @energyweb/exchange-ui-core@1.4.3
 - @energyweb/origin-ui-core@1.14.3
 - @energyweb/origin-ui-irec-core@0.2.2
